config/initializers/cve-2020-8165.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# [CVE-2020-8165] https://groups.google.com/g/rubyonrails-security/c/bv6fW4S0Y1c
+# Can be removed after upgrade to Rails v5.2.4.3+
+
+ActiveSupport::Cache::MemCacheStore.prepend(Module.new do
```

Comment:
Should we also patch Redis Cache?

Reply:
I thought about that, but in the end I don't think we need to do it. `RedisCacheStore` was introduced in 5.2, so it has no meaning in 5.1, just like in 5.0. So as long as we make sure to go straight to 5.2.4.3 when we migrate from 5.1 to 5.2, we're good.